kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -703,6 +705,68 @@ struct RemoveMetadata {
   }
 };
 
+struct GetSTMetadata {
+  const bool read_only = true;
+  std::string description() const {
+    return "getSTMetadata\n"
+           "  Shows State Transfer metadata ";
+  }
+
+  std::string to_json(const bftEngine::bcst::impl::DataStore::CheckpointDesc &chckpDesc,
```

Comment:
Nit: Naming - should be called `toJson()` instead.

Reply:
There was some collision with `concordUtils::toJson()`. This is an internal to the srtuct

Reply:
Can we call it explicitly by `GetSTMetadata::toJson()` and still keep the name?

Reply:
done